### PR TITLE
#163545038 Install dependencies

### PR DESCRIPTION
**Pull request by frankly034**
#### What does this PR do?
Have a basic node app setup with dependencies

#### Description of Task to be completed?
Setup node project
Installed dependencies:
- express
- babel-env
- babel-register
- body-parser
- mocha (globally)

dev-dependencies
- chai
- supertest
- babel-cli
- eslint

set up eslint with airbnb

#### How should this be manually tested?
No manual test set up yet

#### What are the relevant pivotal tracker stories?
#163545038

### CODE REVIEW

**Review by Lumexralph — CHANGES_REQUESTED** [review]
@frankly034 Nice work so far, please ensure to avoid using `console.log()` in your PR ready codebase

**Comment by frankly034** [comment]
Ok, will clean that out. Thanks

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import partyModel from './models/partyModel';
+
+console.log('partiex', partyModel.getAllParty);
```
Comment (Lumexralph) [inline server/app.js]: @frankly034 please remove the `console.log`

Comment (frankly034) [reply server/app.js]: Ok, I'm pushing an update with the changes you requested

Comment (frankly034) [reply server/app.js]: @Lumexralph I've pushed the changes you pointed out

**Review by Lumexralph — APPROVED** [review]
Good Job! This work looks good to me.

**Comment by frankly034** [comment]
Thanks

**Pull request merged**